projects/common-components/src/lib/navigation/navigation-item/navigation-item.component.ts

```diff
@@ -0,0 +1,12 @@
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { FooterItem } from "@jsmu/core";
+
+@Component({
+	selector: "jsmu-navigation-item",
+	templateUrl: "./navigation-item.component.html",
+	styleUrls: ["./navigation-item.component.scss"],
+	changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class NavigationItemComponent {
+	@Input() footerItem!: FooterItem;
```

Comment:
**footer**Item is not correct name, NavigationItem should not be tied to **footer**